pkg/apis/duck/v1alpha1/subscribable_types.go

```diff
@@ -109,7 +109,8 @@ type SubscribableTypeSpec struct {
 
 // SubscribableTypeStatus shows how we expect folks to embed Subscribable in their Status field.
 type SubscribableTypeStatus struct {
-	SubscribableStatus *SubscribableStatus `json:"subscribablestatus,omitempty"`
+	SubscribableStatus   *SubscribableStatus `json:"subscribablestatus,omitempty"`
```

Comment:
```suggestion
	DeprecatedSubscribableStatus   *SubscribableStatus `json:"subscribablestatus,omitempty"`
```

Then leave the next one as `SubscribableStatus`.